src/components/Slider/Slider.tsx

```diff
@@ -141,6 +169,8 @@ const Slider: FC<PropsWithChildren<SliderProps>> = (props) => {
       </div>
     </sliderContext.Provider>
   );
-};
+});
+
+Slider.displayName = 'Slider';
 
 export default Slider;
```

Comment:
please add new slider props to it's story

Reply:
we need this because we are using forwardRef

Reply:
✅